ch00python/036MazeSolution2.ipynb

```diff
@@ -83,14 +81,14 @@
     "for name, room in house.items():\n",
     "    capacity += room['capacity']\n",
     "    occupancy += len(room['people'])\n",
-    "print(\"House can fit {} people, and currently has: {}.\".format(capacity,occupancy))"
+    "print(f\"House can fit {capacity} people, and currently has: {occupancy}.\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As a side note, note how we included the values of `capacity` and `occupancy` in the last line. This is a handy syntax for building strings that contain the values of variables. You can read more about it [here](https://realpython.com/python-string-formatting/#2-new-style-string-formatting-strformat) or in the official do"
+    "As a side note, note how we included the values of `capacity` and `occupancy` in the last line. This is a handy syntax for building strings that contain the values of variables. You can read more about it at this [Python String Formatting Best Practices guide](https://realpython.com/python-string-formatting/#2-new-style-string-formatting-strformat) or in the [official documentation](https://docs.python.org/3.7/library/stdtypes.html?highlight=format#str.format)."
```

Comment:
Should we link to docs for f-strings rather than `str.format`? (although I think it's good to at least mention/link to the latter too somewhere)

Reply:
```suggestion
    "As a side note, note how we included the values of `capacity` and `occupancy` in the last line. This is a handy syntax for building strings that contain the values of variables. You can read more about it at this [Python String Formatting Best Practices guide](https://realpython.com/python-string-formatting/#2-new-style-string-formatting-strformat) or in the [official documentation](https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals)."
```